### PR TITLE
Change execution_config input file from json to yaml

### PR DESCRIPTION
**Pull request by sydhds**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,23 @@
 
 ## Execution config
 
-The default `execution_config.json` contains an example of every available step.
+The default `execution_config.yaml` contains an example of every available step.
 
-## Running the tester
+## Running massa-sc-tester
+
+As execution_config.yaml use a smart contract from massa-sc-examples, you need to build it first:
```
Comment (Eitu33) [inline README.md:11]:
```suggestion
As the default execution_config.yaml uses a smart contract from massa-sc-examples, you need to build it first: 
```

File: execution_config.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+          period: 0
+          thread: 0
+        end:
+          period: 4
+          thread: 16
+    - name: read_main_address_ledger_entry # read balance
+      config:
+        type: read_ledger_entry
+        address: X6Jqk749Dk8=
+    - name: write_new_random_async_message #
```
Comment (Eitu33) [inline execution_config.yaml:50]: Maybe you forgot the comment here?

**Review by Eitu33 — APPROVED** [review]
Very nice 👍

File: README.md (hunk 1 above)
Comment (sydhds) [reply README.md]: Done

File: execution_config.yaml (hunk 2 above)
Comment (sydhds) [reply execution_config.yaml]: Fixed

**Pull request merged**